releasenotes/acs-calling-android-sdk-release-notes.md

```diff
@@ -1,4 +1,13 @@
 # ACS Calling Android (Java) SDK - Release History
+
+## v2.1.0-beta.1 (2022-03-04)
+## New Features:
+- Dominant Speaker Feature: Dominant speakers for a call is an extended feature that allows you to obtain a list of the active speakers in the call. The dominant speakers for a call change dinamically according to their participation in the call.
```

Comment:
Typo? dinamically -> dynamically 
```suggestion
- Dominant Speaker Feature: Dominant speakers for a call is an extended feature that allows you to obtain a list of the active speakers in the call. The dominant speakers for a call change dynamically according to their participation in the call.
```

Reply:
Thanks! Reading it again the word `call` appears four times in a short sentence. Will rephrase it.